pkg/rpc/response/errors.go

```diff
@@ -1,29 +1,35 @@
 package response
 
 import (
-	"errors"
 	"fmt"
-	"net/http"
 )
 
-type (
-	// Error object for outputting JSON-RPC 2.0
-	// errors.
-	Error struct {
-		Code     int64  `json:"code"`
-		HTTPCode int    `json:"-"`
-		Cause    error  `json:"-"`
-		Message  string `json:"message"`
-		Data     string `json:"data,omitempty"`
-	}
-)
+// Error object for outputting JSON-RPC 2.0 errors on the client side.
+type Error struct {
+	Code    int64  `json:"code"`
+	Message string `json:"message"`
+	Data    string `json:"data,omitempty"`
+}
 
-// InternalServerErrorCode is returned for internal RPC server error.
-const InternalServerErrorCode = -32603
+// Various RPC error codes.
+const (
+	// InternalServerErrorCode is returned for internal RPC server error.
+	InternalServerErrorCode = -32603
+	// BadRequestCode is returned on parse error.
+	BadRequestCode = -32700
+	// InvalidRequestCode is returned on invalid request.
+	InvalidRequestCode = -32600
+	// MethodNotFoundCode is returned on unknown method calling.
+	MethodNotFoundCode = -32601
+	// InvalidParamsCode is returned on request with invalid params.
```

Comment:
Maybe it's worth mentioning, that these codes are specified by the JSON-RPC specification.